src/core/mac/sub_mac.cpp

```diff
@@ -412,8 +418,7 @@ void SubMac::ProcessTransmitSecurity(void)
 
 void SubMac::StartCsmaBackoff(void)
 {
-    uint32_t backoff;
-    uint32_t backoffExponent = kCsmaMinBe + mTransmitRetries + mCsmaBackoffs;
+    uint8_t backoffExponent = kCsmaMinBe + mCsmaBackoffs;
```

Comment:
Why is `mTransmitRetries` removed in the formula?

Reply:
The inclusion of `mTransmitRetries` was an enhancement we had in OT only beyond what spec required (and it is used in the case where `SubMac` is taking care of CSMA process and not necessaruly when the radio platform takes care of CSMA process). The SPEC-999 and the new feature added in this PR is defining a similar new common mechnasim for adding delay in between retx so it is replacing this logic.

Reply:
Thanks for clarification. SGTM.